typescript/lib/parsers/openai.ts

```diff
@@ -851,6 +913,31 @@ function multiChoiceMessageReducer(
   return messages;
 }
 
+function buildOutputData(
+  message: Chat.ChatCompletionMessageParam | null
+): OutputData | undefined {
+  let outputData: OutputData | undefined = undefined;
+  if (message !== null) {
+    if (message.content !== null) {
+      outputData = {
+        kind: "string",
+        value: message.content,
+      };
+    } else if (message.function_call !== undefined) {
```

Comment:
Here I am assuming that if we have function call response, then `content` field must be null (`ChatCompletionMessageParam.content` is of type `(string | null)`. I am assuming that both fields can't be set at the same time